docs/tips-tricks.md

```diff
@@ -9,7 +9,79 @@ If the VPN connection fails or the container for any other reason loses connecti
 
 #### Let other containers use VPN
 
-[TODO](https://github.com/haugene/docker-transmission-openvpn/issues/1558): Relevant issues...
+To let other containers use VPN you have to add them to the same Service network as your VPN container runs, you can do this by adding `network_mode: "service:transmission-openvpn"`. Additionally you have to set `depends_on` to the `transmission-openvpn` service to let docker-compose know that your new container should start **after** `transmission-openvpn` is up and running. As final step you can add `healthcheck` to you service.
+
+As an example, lets add [Jackett](https://github.com/linuxserver/docker-jackett) to the `transmission-openvpn` network based on example from [Running the container](run-container.md):
+
+```yaml
+version: '3.3'
+services:
+    transmission-openvpn:
+        cap_add:
+            - NET_ADMIN
+        volumes:
+            - '/your/storage/path/:/data'
+        environment:
+            - OPENVPN_PROVIDER=PIA
+            - OPENVPN_CONFIG=france
+            - OPENVPN_USERNAME=user
+            - OPENVPN_PASSWORD=pass
+            - LOCAL_NETWORK=192.168.0.0/16
+        logging:
+            driver: json-file
+            options:
+                max-size: 10m
+        ports:
+            - '9091:9091'
+            - '9117:9117'  # This is Jackett Port – managed by VPN Service Network
+        image: haugene/transmission-openvpn
+    jackett:
+        image: lscr.io/linuxserver/jackett:latest
+        container_name: jackett
+        environment:
+            - PUID=1000
+            - PGID=1000
+            - TZ=Europe/London
+            - AUTO_UPDATE=true #optional
+            - RUN_OPTS=<run options here> #optional
+        volumes:
+            - <path to data>:/config
+            - <path to blackhole>:/downloads
+        # You have to comment ports, they should be managed in transmission-openvpn section now.
+#       ports:
+#           - 9117:9117
+        restart: unless-stopped
+        network_mode: "service:transmission-openvpn" # Add to the transmission-openvpn Container Network
+        depends_on:
+            - transmission-openvpn # Set dependency on transmission-openvpn Container
+        healthcheck: # Here you will check if transmission is reachable from the Jackett container via localhost
+            test: curl -f http://localhost:9091 || exit 1
+            # Use this test if you protect your transmission with user and password 
+            # comment test above and un-comment line below.
+            #test: curl -f http://${TRANSMISSION_RPC_USERNAME}:${TRANSMISSION_RPC_PASSWORD}@localhost:9091 || exit 1
+            interval: 5m00s
+            timeout: 10s
+            retries: 2
+            start_period: 30s
+```
+
+##### Check if container is using VPN
+
+After container starts, simply call `curl` under your it to check your IP Address. E.g. for example with Jackett you will should your VPN IP address:
```

Comment:
> call `curl` under your it to check

> with Jackett you will should your VPN IP

typos?

Reply:
You totally right... I did some rewording and than Joda attacks me. I already removed this branch, and did update in a new one https://github.com/haugene/docker-transmission-openvpn/pull/2259/files#diff-94fc4b2749c1284cb8a16125e911aed6f7dc55395b44aabed1dd738eac5bb505R69